README.md

```diff
@@ -29,6 +29,9 @@ bip39.mnemonicToSeedHex('basket actual')
 bip39.mnemonicToSeed('basket actual')
 // => <Buffer 5c f2 d4 a8 b0 35 5e 90 29 5b df c5 65 a0 22 a4 09 af 06 3d 53 65 bb 57 bf 74 d9 52 8f 49 4b fa 44 00 f5 3d 83 49 b8 0f da e4 40 82 d7 f9 54 1e 1d ba 2b ...>
 
+bip39.mnemonicToSeed('basket actual', 'a password')
```

Comment:
Can the seed derived from the mnemonic plus password also be displayed as a mnemonic?

Reply:
bitcore-mnemonic supports this in their implementation of BIP39 - And I do believe it is part of the BIP39 spec, is it not?
```
// bitcore-mnemonic
var mnemonic = new Mnemonic(256, Mnemonic.Words.ENGLISH);
var seed = mnemonic.toSeed("password");
var encryptedMnemonic = Mnemonic.fromSeed(seed, Mnemonic.Words.ENGLISH);
```

I am unable to create the same result in bip39.js

Reply:
That isn't a standard way to do things,  no.

Reply:
That is a very strange usage of BIP39,  and isn't supported by this library.

Reply:
@junderw you might have experience here

Reply:
@lacksfish That is not the correct usage of BIP39.

Reply:
`Mnemonic.fromSeed` is poorly named in their library. The "seed" referred to here is random bytes of entropy.

Whereas the "seed" referred to in the Mnemonic.prototype.toSeed is the result of the pbkdf2 hash to be used as a seed for BIP32.

It's confusing, understandably... but the way @lacksfish is using it is not proper.

Reply:
Random bytes of entropy is the same as a randomly created mnemonic. It is both (hopefully) random data.

Using random data (first mnemonic) to derive more pseudo-random data (second mnemonic hidden by password) shouldn't be much of a concern. I agree though, the 256 entropy limit will not allow for this feature within the scope of revised the BIP39 spec. :(

@junderw It may not be the intended use of BIP39, but I'm interested in the feature for plausible-deniability.